spk/minio/src/wizard/install_uifile

```diff
@@ -0,0 +1,67 @@
+[
+	{
+		"step_title": "minio configuration",
+		"items": [
+		  {
+			"type": "textfield",
+			"desc": "Data directory location",
```

Comment:
Only the first subitem is called "data directory location". To remove this additional text, remove the "desc" property and the subitems are displayed only ("type": "textfield" is required)

Reply:
Updated in latest version